ports/unix/Makefile

```diff
@@ -144,21 +169,27 @@ ifeq ($(MICROPY_SSL_AXTLS),1)
 endif
 endif
 
-# If the variant enables it, enable modbluetooth.
 ifeq ($(MICROPY_PY_BLUETOOTH),1)
 ifeq ($(MICROPY_BLUETOOTH_BTSTACK),1)
-HAVE_LIBUSB := $(shell (which pkg-config > /dev/null && pkg-config --exists libusb-1.0) 2>/dev/null && echo '1')
 
-# Figure out which BTstack transport to use.
-ifeq ($(HAVE_LIBUSB),1)
-# Default to btstack-over-usb.
-MICROPY_BLUETOOTH_BTSTACK_USB ?= 1
-else
-# Fallback to HCI controller via a H4 UART (e.g. Zephyr on nRF) over a /dev/tty serial port.
-MICROPY_BLUETOOTH_BTSTACK_H4 ?= 1
+ifndef MICROPY_BLUETOOTH_BTSTACK_HCI_DUMP
```

Comment:
Please add a comment here about what this value is for, similar to the other config items.

Suggest even moving it up with the other user-settable values.